src/Auth0.OidcClient.WinForms/WebViewBrowser.cs

```diff
@@ -0,0 +1,59 @@
+﻿using IdentityModel.OidcClient.Browser;
+using Microsoft.Toolkit.Forms.UI.Controls;
+using System;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Auth0.OidcClient
+{
+    public class WebViewBrowser : IBrowser
```

Comment:
Docblock/comment?

Reply:
Seems like a lot of this is duplicated from `src/Auth0.OidcClient.WPF/WebViewBrowser.cs` ...

Reply:
It is duplicated but I can't move it into the shared library because iOS and Android do not support WebView.